R/mathemtical_equations.R

```diff
@@ -1,73 +1,128 @@
-SEIR_nonVD_equation <- c(r"(    &\frac{dS}{dt} = - \beta \frac{ S I}{N^q}\\)", 
-                         r"(    &\frac{dE}{dt} = \beta \frac{ S I}{N^q} - \gamma {E}\\)", 
-                         r"(    &\frac{dI}{dt} = \gamma {E} - \sigma {I}\\)", 
-                         r"(    &\frac{dR}{dt} = \sigma {I}\\)", 
-                         r"(    &R_0 =  \frac{\beta}{\gamma} S(0)^q\\)", 
-                         r"(    &\begin{cases} 1 & \text{, frequency-dependent} \\\\ 0 & \text{, density-dependent} \end{cases}\\)")
-# SEIR model with vital dynamic
-SEIR_VD_equation <- c(r"(    &\frac{dS}{dt} = \mu_B {N} - \mu_D {S} - \beta \frac{ S I}{N^q}\\)", 
-                      r"(    &\frac{dE}{dt} = \beta \frac{ S I}{N^q} - \gamma {E} - - \mu_D {E}\\)", 
-                      r"(    &\frac{dI}{dt} = \gamma {E} - \sigma {I} - \mu_D {I}\\)", 
-                      r"(    &\frac{dR}{dt} = \sigma {I}) - \mu_D {R}\\)",  
-                      r"(    &R_0 =  \frac{\beta}{\gamma} S(0)^q\\)", 
-                      r"(    &\begin{cases} 1 & \text{, frequency-dependent} \\\\ 0 & \text{, density-dependent} \end{cases}\\)")
 
-# SEIRD model with no vital dynamic
-SEIRD_nonVD_equation <- c(r"(    &\frac{dS}{dt} = - \beta \frac{ S I}{N^q}\\)", 
-                          r"(    &\frac{dE}{dt} = \beta \frac{ S I}{N^q} - \gamma {E}\\)",
-                          r"(    &\frac{dI}{dt} = \gamma {E} - \sigma {I} - \delta {I}\\)",
-                          r"(    &\frac{dR}{dt} = \sigma {I}\\)",  
-                          r"(    &\frac{dD}{dt} = \delta {I}\\)",
-                          r"(    &R_0 =  \frac{\beta}{\gamma} S(0)^q\\)", 
-                          r"(    &\begin{cases} 1 & \text{, frequency-dependent} \\\\ 0 & \text{, density-dependent} \end{cases}\\)")
-# SEIRD model with vital dynamic
-SEIRD_VD_equation <- c(r"(    &\frac{dS}{dt} = \mu_B {N} - \mu_D {S} - \beta \frac{ S I}{N^q}\\)",
-                       r"(    &\frac{dE}{dt} = \beta \frac{ S I}{N^q} - \gamma {E} - - \mu_D {E}\\)",
-                       r"(    &\frac{dI}{dt} = \gamma {E} - \sigma {I} -  \delta {I} - \mu_D {I}\\)", 
-                       r"(    &\frac{dR}{dt} = \sigma {I}) - \mu_D {R}\\)", 
-                       r"(    &\frac{dD}{dt} = \delta {I}\\)", 
-                       r"(    &R_0 =  \frac{\beta}{\gamma} S(0)^q\\)", 
-                       r"(    &\begin{cases} 1 & \text{, frequency-dependent} \\\\ 0 & \text{, density-dependent} \end{cases}\\)")
\ No newline at end of file
+SIRS <- function(mu,forceOfInfection)
+{
+  if (!mu)
+  {
+    S = paste0(Susceptible, "- ", forceOfInfection, r"(+ \xi{R}\\)")
+    I = paste0(Infection, forceOfInfection, r"(- \gamma {I}\\)")
+    R = paste0(Recover, r"(\gamma {I} - \xi{R}\\)" )
+  } else 
+  {
+    S = paste0(Susceptible, r"( \mu{N} -)", forceOfInfection, r"(+ \xi{R} - \nu{S}\\)")
+    I = paste0(Infection, forceOfInfection, r"(- \gamma {I} - \nu{I}\\)" )
+    R = paste0(Recover, r"(\gamma {I} - \xi{R} - \nu{R}\\)")
```

Comment:
$$S = \mu_B N - \beta\frac{SI}{N} + \xi R - \mu_D S$$

`\mu_D S`, not `\nu S`.

The same issue appears in $I$ and $R$.